person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, age
+
+  def initialize(type, age, _parent_permission, name = 'unkown')
```

Comment:
- [ ] Kindly remove type as a parameter. The type is not needed because your `can_use_services?` method should check if the `of_age?` returns true or `parent_permission` returns true.